kyuubi-server/src/main/scala/org/apache/kyuubi/client/KyuubiSyncThriftClient.scala

```diff
@@ -43,7 +43,8 @@ class KyuubiSyncThriftClient private (
     protocol: TProtocol,
     engineAliveProbeProtocol: Option[TProtocol],
     engineAliveProbeInterval: Long,
-    engineAliveTimeout: Long)
+    engineAliveTimeout: Long,
+    engineAliveCloseConnection: Boolean = false)
```

Comment:
Setting a default value here for better compatibility.

Reply:
seems default value is not needed

Reply:
Sure, I will delete it!